phpcs-ruleset.xml

```diff
@@ -37,6 +37,9 @@
     <!-- This module is not supported by Core team so we exclude it from QA -->
     <exclude-pattern>profiles/*/modules/custom/nexteuropa_newsroom/</exclude-pattern>
 
+    <!-- This module is not supported by Core team so we exclude it from QA, to avoid phpcs issue -->
+    <exclude-pattern>profiles/*/modules/custom/nexteuropa_newsroom/</exclude-pattern>
+
```

Comment:
I see two issues here:

1. this is out of scope
2. it's duplicated

Maybe a wrong merge ?

Reply:
Nice catch Pol.
We will have to remove that.